examples/webgl_loader_ifc.html

```diff
@@ -73,47 +76,23 @@
 
 				//Setup IFC Loader
 				const ifcLoader = new IFCLoader();
-				ifcLoader.ifcManager.setWasmPath( 'jsm/loaders/ifc/' );
+				await ifcLoader.ifcManager.setWasmPath( 'https://unpkg.com/web-ifc@0.0.36/', true );
+
+				await ifcLoader.ifcManager.parser.setupOptionalCategories({
+					[IFCSPACE]: false,
+				});
+
+				await ifcLoader.ifcManager.applyWebIfcConfig({
+					USE_FAST_BOOLS: true
+				});
```

Comment:
Very small nit: lint issues here. Should have spaces inside the parens and square brackets.